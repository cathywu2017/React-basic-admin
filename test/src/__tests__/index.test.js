import Index from '../index';

it('renders index correctly', () => {
  expect(JSON.stringify(Index)).toMatchSnapshot();
});
