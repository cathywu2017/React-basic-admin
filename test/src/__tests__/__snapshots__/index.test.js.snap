// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders index correctly 1`] = `"{}"`;
